db_init.sql

```diff
@@ -0,0 +1,6 @@
+CREATE USER has_many_user;
```

Comment:
The user and database should be created by setting the correct values in your .env file.